pkgs/development/libraries/harfbuzz/default.nix

```diff
@@ -4,7 +4,7 @@
 , pkg-config
 , glib
 , freetype
-, cairo
+, fontconfig
```

Comment:
HarfBuzz does not have a direct dependency on FontConfig (the configure script used to check for it optionally, but it was actually unused).

Reply:
Thank you for fixing the configure script _and_ pointing this out. :+1: 

I only had to update it to remove `fontconfig`.